README.md

```diff
@@ -1,6 +1,17 @@
 API Application for camptocamp.org v6
 =====================================
 
+Requirements
+------------
+
+ * Python 3.4
+ * PostgreSQL 9.4 / PostGIS 2.1
+ * virtualenv (1.7 or higher)
```

Comment:
OK thanks.
I have added those components to the list at 727756d
Not sure what packages should be installed though.